cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -112,6 +117,9 @@ private KafkaCluster(String namespace, String cluster, Labels labels) {
         this.metricsConfigVolumeName = "kafka-metrics-config";
         this.metricsConfigMountPath = "/opt/prometheus/config/";
 
+        this.logConfigVolumeName = "kafka-log-config";
+        this.logConfigMountPath = "/opt/kafka/logconfig/";
```

Comment:
Can we map the `log4j.properties` file over the existing file to make it more clear which file is actually being used?

Reply:
Yes, I had some issue with this, so I just fast workaround-ed and forgot to fix that.

Reply:
Ok, fine with me if there are some issues.